src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -66,12 +66,19 @@ export class ZerionBalancesApi implements IBalancesApi {
       const cacheDir = CacheRouter.getZerionBalancesCacheDir(args);
       const chainName = this._getChainName(args.chainId);
       const currency = args.fiatCode.toLowerCase();
-      const url = `${this.baseUri}/v1/wallets/${args.safeAddress}/positions?filter[chain_ids]=${chainName}&currency=${currency}&sort=value`;
+      const url = `${this.baseUri}/v1/wallets/${args.safeAddress}/positions`;
       const { data } = await this.dataSource.get<ZerionBalances>({
         cacheDir,
         url,
         notFoundExpireTimeSeconds: this.defaultNotFoundExpirationTimeSeconds,
-        networkRequest: { headers: { Authorization: `Basic ${this.apiKey}` } },
+        networkRequest: {
+          headers: { Authorization: `Basic ${this.apiKey}` },
+          params: {
+            'filter[chain_ids]': chainName,
+            currency,
```

Comment:
nitpick: a refactor on the name of this property would result in the wrong request being done. While the shorthand version is convenient this is something that we should pay attention to in case we change it in the future.

Reply:
Yeah, totally agree. I have another PR in the making to use `NetworkRequest.params` on `valk-balances-api.service.ts` as well, so I'll make the change there, thanks!